libconglomerate.user.js

```diff
@@ -576,6 +576,7 @@ function safebuttons() {
     if (getSetting('safety-extra-well') == 'true') { safetyButtons("//form[@name='skilluse']/input[@type='submit' and starts-with(@value, 'Open Arcane Well')]", 'disable'); }
     if (getSetting('safety-extra-mark') == 'true') { safetyButtons("//form[@name='skilluse']/input[@type='submit' and starts-with(@value, 'Create Nexal Mark')]", 'disable'); }
     if (getSetting('safety-extra-hself') == 'true') { safetyButtons("//form[@name='skilluse']/input[@type='submit' and starts-with(@value, 'Heal Self')]", 'disable'); }
+    if (getSetting('safety-extra-shape') == 'true') { safetyButtons("//form[@name='spellaaoettack']/input[@type='submit' and @value='Shape Area Spell (5 AP)']", 'disable'); }
```

Comment:
I recommend using the `starts-with` directive and removing the `(5 AP)` cost string, as with the other safeties.
Doing so will make the code less brittle (i.e. not breaking if the developers change the cost of the ability)